packages/protocol/test-sol/devchain/e2e/utils.sol

```diff
@@ -1,52 +1,103 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity >=0.8.7 <0.8.20;
 
-import "@celo-contracts/common/interfaces/IRegistry.sol";
-import { IEpochManager } from "@celo-contracts/common/interfaces/IEpochManager.sol";
+// Test imports
+import { TestWithUtils08 } from "@test-sol/TestWithUtils08.sol";
+
+// Celo contracts imports
 import { IAccounts } from "@celo-contracts/common/interfaces/IAccounts.sol";
+// import { BlockchainParameters } from "@celo-contracts/governance/BlockchainParameters.sol";
+import { ICeloUnreleasedTreasury } from "@celo-contracts/common/interfaces/ICeloUnreleasedTreasury.sol";
+import { ICeloToken } from "@celo-contracts/common/interfaces/ICeloToken.sol";
+// import { DoubleSigningSlasher } from "@celo-contracts/governance/DoubleSigningSlasher.sol";
```

Comment:
<img width="713" alt="Screenshot 2025-05-20 at 14 23 04" src="https://github.com/user-attachments/assets/26303621-a6a9-464a-bac0-a3e823e87f00" />

Reply:
Not sure why these were added since they are not being used in this file.

Also note:
`DownTimeSlasher`,`DoubleSigningSlasher`,`BlockchainParameters` have been deprecated post L2.